### DISCUSSION TITLE
An error occurred to change the password

### DISCUSSION THREAD

**Discussion by Dao-Tek** [nasirkhan/laravel-starter#594] [General]
Hello！ What happens when I change the user password with the following error? What should have gone wrong.
`Illuminate\Http\Request::Illuminate\Foundation\Providers\{closure}(): Argument #1 ($rules) must be of type array, Illuminate\Http\Request given, called in 
.../vendor/laravel/framework/src/Illuminate/Macroable/Traits/Macroable.php on line 123`

**Comment by nasirkhan** [comment]
thanks for reporting the issue. 
A patch has been created and merged to the main branch.
